packages/e2e/test/web-extension/tsconfig.json

```diff
@@ -25,6 +25,15 @@
     },
     {
       "path": "../../../wallet/src"
+    },
+    {
+      "path": "../../../cardano-services-client/src"
```

Comment:
Should add it to `package.json` dependencies too

Reply:
The dependency was added here: https://github.com/input-output-hk/cardano-js-sdk/blob/10b09f7c68878809e3b8898bd08abc104a706c9c/packages/e2e/package.json#L61

Reply:
We should probably remove it from web-extension tsconfig.json too (move to e2e package tsconfig)

Reply:
addressed here: https://github.com/input-output-hk/cardano-js-sdk/pull/356/commits/7df0b700f623e8cddf629e9c20c5ee619b3c6c3d it was already in the e2e package tsconfig :)

Reply:
I think [ogmios reference](https://github.com/input-output-hk/cardano-js-sdk/pull/356/files#diff-cf4164ff5ad6c088c8d5aaaa7244bb83f76ae7106d5abbeb59861d2691748943R32-R33) should also be removed then

Reply:
addressed here: https://github.com/input-output-hk/cardano-js-sdk/pull/356/commits/608ba40c8e1327a59eaa81bba4d94f32523d08a8 (core was also duplicated)